### PR TITLE
Add pages for breadcrumbs and tabs

### PR DESCRIPTION
**Pull request by chaowentan**
Also remove periods according to the Material Design writing guideline

image

image

### CODE REVIEW

File: kolibri/plugins/style_guide/templates/style_guide/style_guide.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (indirectlylit) [inline kolibri/plugins/style_guide/templates/style_guide/style_guide.html]: unfortunately we can't introduce dependencies on the internet

Search our code for `mat-svg` to see an alternate way of including material design icons

Comment (chaowentan) [reply kolibri/plugins/style_guide/templates/style_guide/style_guide.html]: Unfortunately, `<mat-svg>` seems to rely on a WebPack preloader called "svg-icon-inline-loader", which won't be able to work with the stringified code example we have in the style guide.

And I thought the style guide will be used internally (at least for now), so having external dependency shouldn't be a problem?

Comment (indirectlylit) [reply kolibri/plugins/style_guide/templates/style_guide/style_guide.html]: I think we still want to allow devs to work offline.

Actually we already have [material icons](https://www.npmjs.com/package/material-design-icons) as a global NPM dependency, so the font should already be available. @christianmemije might be able to assist

Comment (chaowentan) [reply kolibri/plugins/style_guide/templates/style_guide/style_guide.html]: Ah, ok. I didn't know the Material Design fonts are already installed. So I've changed my PR to use that instead. Please take another look.

**Comment by indirectlylit** [comment]
down with `.`s!

**Pull request merged**